lib/internal/util.js

```diff
@@ -179,3 +180,18 @@ exports.toLength = function toLength(argument) {
   const len = toInteger(argument);
   return len <= 0 ? 0 : Math.min(len, Number.MAX_SAFE_INTEGER);
 };
+
+const OS_SIGNALS_TO_NAMES_MAPPING = Object.create(null);
+Object.keys(OS_SIGNALS).forEach((signal) => {
```

Comment:
Maybe we should create this mapping lazily. It might help to switch to a `for` loop to speed things up too.